README.md

```diff
@@ -1 +1,35 @@
-# lab-07B
\ No newline at end of file
+# lab-07B
+**Project Name**
+**Author:** Randee Orion
+**Links and Resources**
+
+[gitHub PR](https://github.com/randee-401-advanced-javascript/lab-07B/pull/1)
+[gitHub Actions](https://github.com/randee-401-advanced-javascript/lab-07B/tree/master/.github/workflows)
```

Comment:
https://github.com/randee-401-advanced-javascript/lab-07B/pull/2/checks - this is the link you needed